src/modules/Auth/hooks/useSignInUser.ts

```diff
@@ -1,9 +1,30 @@
+import { useHistory } from 'react-router-dom';
+import { HOME_PAGE } from '@/constants/routes';
 import { useSignInUserMutation } from '@/queries/auth';
+import { SignInUserData } from '@/types/auth';
 
-export function useSignInUser() {
-  return useSignInUserMutation({
-    onSuccess: () => {
-      // Store a token to the local storage
+type UseCreateUserParams = {
```

Comment:
The name of parameters pattern is `{name of hook}Params`
```suggestion
type UseSignInUserParams = {
```